docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
Yeah, I think we can have 2 ways of change detection: event-based and predicate-based.
1) `<q-grid:td on='edit'>` - similar to ng-switch, but invoked only on predefined events.
2)  `<q-grid:td when="$mode == 'edit'">` - bound to digest loop to track all changes.

I can see one problem with second option. Predicates are not guaranteed to be mutual exclusive. And we can have a case when more then one template is suitable.